### PR TITLE
UIEH-1046: Resource: Re-order accordions display

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Description
Reorder accordions in Resource show page

## Coverage
image

## Issues
[UIEH-1046](https://issues.folio.org/browse/UIEH-1046)

### CODE REVIEW

File: src/components/resource/components/resource-information/resource-information.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        <Headline size="large" tag="h3">
+          <FormattedMessage id="ui-eholdings.resource.resourceInformation" />
+        </Headline>
+      )}
+      open={isOpen}
+      id="resourceShowInformation"
+      onToggle={onToggle}
+    >
+      <KeyValueColumns>
+        <div>
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/components/resource-information/resource-information.js:44]: Do we need this `<div>`?

File: src/components/resource/components/resource-information/resource-information.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+              </div>
+            </KeyValue>
+          )}
+        </div>
+        <div>
+          <KeyValue label={<FormattedMessage id="ui-eholdings.label.package" />}>
+            <div data-test-eholdings-resource-show-package-name>
+              <InternalLink
+                to={`/eholdings/packages/${model.packageId}`}
+              >
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/components/resource-information/resource-information.js:126]: Please, write single attribute on the same line with tag name
```suggestion
              <InternalLink to={`/eholdings/packages/${model.packageId}`} >
```

File: src/components/resource/components/resource-information/resource-information.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                <FormattedMessage id="ui-eholdings.package.addToken" />
+              </Button>
+            </KeyValue>
+          )}
+
+          <KeyValue label={<FormattedMessage id="ui-eholdings.label.provider" />}>
+            <div data-test-eholdings-resource-show-provider-name>
+              <InternalLink
+                to={`/eholdings/providers/${model.providerId}`}
+              >
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/components/resource-information/resource-information.js:148]:
```suggestion
              <InternalLink to={`/eholdings/providers/${model.providerId}`} >
```

File: src/components/resource/components/resource-settings/resource-settings.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+          }
+        </div>
+      </KeyValue>
+
+      {
+        hasInheritedProxy && (
+          !proxyTypes.request.isResolved || model.isLoading
+            ? (
+              <Icon icon="spinner-ellipsis" />
+            )
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/components/resource-settings/resource-settings.js:82]:
```suggestion
            ? <Icon icon="spinner-ellipsis" />
```

File: src/components/resource/components/resource-settings/resource-settings.js [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+        )
+      }
+
+      {model.url && (
+        <KeyValue label={model.title.isTitleCustom ?
+          <FormattedMessage id="ui-eholdings.custom" />
+          :
+          <FormattedMessage id="ui-eholdings.managed" />}
+        >
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/components/resource-settings/resource-settings.js:98]:
```suggestion
        <KeyValue label={model.title.isTitleCustom
          ? <FormattedMessage id="ui-eholdings.custom" />
          : <FormattedMessage id="ui-eholdings.managed" />}
        >
```

File: src/components/resource/components/resource-settings/resource-settings.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+          {haveAccessTypesLoaded
+            ? (
+              <AccessTypeDisplay
+                accessTypeId={getAccessTypeId(model)}
+                accessStatusTypes={accessStatusTypes}
+              />
+            )
+            : (
+              <Icon icon="spinner-ellipsis" />
+            )}
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/components/resource-settings/resource-settings.js:120]:
```suggestion
            : <Icon icon="spinner-ellipsis" />
           }
```

File: src/components/resource/resource-edit.js [hunk 7]
```diff
@@ -141,16 +141,20 @@ export default class ResourceEdit extends Component {
           const { model } = this.props;
           const { customCoverages: customCoverageDateValues } = values;
           let coverageDates = model.managedCoverages;
-          if (customCoverageDateValues && customCoverageDateValues.length > 0) {
+
+          const customCoverageExists = customCoverageDateValues && customCoverageDateValues.length > 0;
+          if (customCoverageExists) {
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/resource-edit.js:146]: Empty line after declaration
```suggestion
          const customCoverageExists = customCoverageDateValues && customCoverageDateValues.length;
    
          if (customCoverageExists) {
```

File: src/components/resource/components/holding-status/holding-status.test.js [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+describe('Given HoldingStatus', () => {
+  let component;
+  const onToggleMock = jest.fn();
+  const onAddToHoldingsMock = jest.fn();
+
+  const defaultModel = {
+    update: {
+      isPending: false,
+      changedAttributes: {}
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/components/holding-status/holding-status.test.js:20]:
```suggestion
      changedAttributes: {},
```

File: src/components/resource/components/holding-status/holding-status.js [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+            : (
+              <Headline margin="none">
+                {
+                  resourceSelected
+                    ? (
+                      <FormattedMessage id="ui-eholdings.selected" />
+                    )
+                    : (
+                      <FormattedMessage id="ui-eholdings.notSelected" />
+                    )
```
Comment (Vladyslav-Velytskyi) [inline src/components/resource/components/holding-status/holding-status.js:73]:
```suggestion
                    ? <FormattedMessage id="ui-eholdings.selected" />
                    : <FormattedMessage id="ui-eholdings.notSelected" />
```

File: src/components/access-type-display/access-type-display.js [hunk 10]
```diff
@@ -1,20 +1,38 @@
...
+  if (!accessStatusTypes?.items?.data?.length) {
+    return null;
+  }
+
+  const formattedAccessTypes = getAccessTypeIdsAndNames(accessStatusTypes.items.data);
+
+  const selectedAccessType = formattedAccessTypes
     .find(accessType => accessType.id === accessTypeId)?.name || '';
 
+
```
Comment (Vladyslav-Velytskyi) [inline src/components/access-type-display/access-type-display.js:35]: This is unnecessary changes

**Review by Vladyslav-Velytskyi — APPROVED** [review]

**Review by Yuliia-Dovhal — APPROVED** [review]
